portworx-Cassandra/1.8/README.md

```diff
@@ -0,0 +1,73 @@
+
+The following instruction will install Cassandra service on DC/OS cluster backed by PX volumes for persistent storage.
+
+
+# Prerequisites
+
+- A running DC/OS v1.8 cluster with at least 3 private agents with Portworx running on all three
+- Portworx works best when installed on all nodes in a DC/OS cluster.  If Portworx is to be installed on a subset of the cluster, then:
+  * the agent-nodes must include attributes indicating the participate in the Portworx cluster.
```

Comment:
No need to mention this. Just mention that if PX is not installed on all nodes then constraints must be used to specify the nodes where PX is installed